firefox-ios/Client/ContentBlocker/ContentBlocker.swift

```diff
@@ -215,13 +246,17 @@ class ContentBlocker {
 // ruleStore.
 extension ContentBlocker {
     private func loadJsonFromBundle(forResource file: String, completion: @escaping (_ jsonString: String) -> Void) {
-        DispatchQueue.global().async { [weak self] in
-            guard let path = Bundle.main.path(forResource: file, ofType: "json"),
-                  let source = try? String(contentsOfFile: path, encoding: .utf8)
-            else {
-                self?.logger.log("Error unwrapping the resource contents", level: .warning, category: .webview)
-                assertionFailure("Error unwrapping the resource contents")
-                return
+        let logger = self.logger
+        DispatchQueue.global().async {
+            var source = ""
+            do {
+                let fileTrimmed = file.hasSuffix(".json") ? String(file.dropLast(5)) : file
```

Comment:
Can we do something about this magic number? We should at least add a comment I think.

Reply:
This is hardly a magic number 😄 (there's nothing particularly mysterious about a file suffix string length). That said I'm happy to pull it out and call into `count` instead; I would hope that would be optimized away by the Swift compiler but even if not it won't impact performance. I'll add a comment also.